### DISCUSSION TITLE
send logs to elasticsearch

### DISCUSSION THREAD

**Discussion by marefati110** [strapi/strapi#6846] [Legacy Topics]
hi 
i want to send strapi logs into elasticsearch 
what is the best practice ??
and how can save logs as file?

**Comment by omatviiv** [comment]
+1

**Comment by derrickmehaffy** [comment]
I would imagine it would be similar to Sentry, [here is our guide on that](https://strapi.io/documentation/v3.x/guides/error-catching.html), so you will probably want to find a package to hook into ES and intercept the messages to pass along
